src/System.IO.Pipes/tests/PipeTestBase.cs

```diff
@@ -80,7 +80,11 @@ public void Dispose()
         /// <returns></returns>
         protected static string GetUniquePipeName()
         {
-            return Path.GetRandomFileName();
+            return
+#if uap
```

Comment:
actually there is a better way to do it, instead of partial files I'll do PlatformDetection